src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,8 +18,10 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Parameters: " + PREFIX_KEYWORD + "KEYWORD [" + PREFIX_KEYWORD + "MORE_KEYWORDS] "
```

Comment:
Just a minor point but perhaps could add "..." after [MORE_KEYWORDS]! To suggest that user can input more than 1 more keywords

Reply:
Thanks for reviewing @lzan98 !! :))

Let me add the `...` after `[MORE_KEYWORDS]`. 

Though I'm not very sure what you mean by the previous comment? As the updated functionality in this PR is such that the user can input multiple keywords using the `k/` prefix. For example for the command `find k/alex yeoh k/bernice f/name` will return persons whose name contain either `alex yeoh` or `bernice`. Hence it does allow for multiple prefixes! :)

Reply:
No problem! Let's wait for 1 more person to review and we can merge this.

And sorry, I meant prefix field instead! e.g. Are we supporting `find k/alex yeoh k/E0123456 f/name f/studentID`? Think we discussed about it but I forgot. If no, then perhaps we could make sure there is only one prefix field! Let's discuss this again on Sunday!